src/main/java/seedu/duke/reflection/GetCommand.java

```diff
@@ -0,0 +1,155 @@
+package seedu.duke.reflection;
+
+import seedu.duke.command.Command;
+import seedu.duke.exception.BadCommandException;
+import seedu.duke.exception.InvalidCommandException;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Random;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class GetCommand extends Command {
+    private static final int NUM_OF_RANDOM_QUESTIONS = 5;
+    private static final String FEATURE_NAME = "Self Reflection";
+    private static final String COMMAND_KEYWORD = "get";
+    private static final String FULL_DESCRIPTION = "";
+    private static final String ARGUMENT = "get";
+    private static final String PAYLOAD = "";
+    private static final int ARGUMENT_PAYLOAD_SIZE = 1;
+    private static final String INVALID_COMMAND_MSG = "Command is invalid.";
+    private static final String INVALID_COMMAND_NOTES = "Please check the available commands "
+            + "and the format of commands.";
```

Comment:
Should the + sign be underneath the Please?

Reply:
I think it's 8 spaces from the previous line, @haoyangw can help double check this?

Reply:
yes, confirmed just now 8 spaces from previous line